src/helpers.ts

```diff
@@ -223,7 +239,9 @@ export function getMinterDetails<T>(config: T): TypedMap<string, JSONValue> {
   if (
     !(config instanceof ProjectMinterConfiguration || config instanceof Minter)
   ) {
-    throw new Error("cannot find extra minter details on " + config);
```

Comment:
this is weird, but for some reason the changes in this PR no longer allow the previous code. Kind of a code smell, but I can't figure out why this needed to be changed...

Reply:
Does this work? 

```
`cannot find extra minter details on {config}`
```

Reply:
assuming you meant with the dollar sign, it doesn't work:
```
`cannot find extra minter details on ${config}`
```

I think this is actually because of some change in ordering of `handleSetMinterDetailsGeneric`, which improves our robustness by not calling this function with a generic type. This probably would have crashed the subgraph before if we had sent over something that was not a `ProjectMinterConfiguration` or `Minter`, and that didn't have a `toString()` method 💡